### PR TITLE
RGB stream stability before SPECIAL FRAME

### PR DESCRIPTION
**Pull request by maloel**

### CODE REVIEW

File: src/l500/ac-trigger.h [hunk 1]
```diff
@@ -62,6 +62,11 @@ namespace ivcam2 {
         double _last_temp = 0;
         std::shared_ptr< temp_check > _temp_check;
 
+        class rgb_stability;
```
Comment (aangerma) [inline src/l500/ac-trigger.h]: remove

File: src/l500/ac-trigger.h [hunk 2]
```diff
@@ -62,6 +62,11 @@ namespace ivcam2 {
...
+        bool _need_to_wait_for_color_sensor_stability = false;
```
Comment (aangerma) [inline src/l500/ac-trigger.h:65]: Shouldn't it be atomic?

Comment (maloel) [reply src/l500/ac-trigger.h:65]: If it does need to be, then we have other variables that need to be. After looking at the code, I think it's fairly safe and that the only sets are at the end-points (trigger/stop) and from set_color_frame. The one complication would be stop & set_color_frame happening together -- but they both set it to false, so I don't see how they'd conflict.

File: src/l500/ac-trigger.h (hunk 1 above)
Comment (maloel) [reply src/l500/ac-trigger.h]: Removed

**Pull request merged**